docs/deploy.md

```diff
@@ -34,3 +34,25 @@ Note also that Whippet, by default, will not deploy your application if the comm
 ```
 $ whippet deploy -f /var/local/myapp
 ```
+
+### Deploying the `public/` directory
+
+By default, the `public/` directory inside a WordPress app will be copied into
+the app directory on deploy.
+
+By passing in the `-p` or `--public` argument, `public/` will be deployed in
+an adjacent directory to the app itself, i.e.
+
+```
+$ whippet deploy -p /var/local/myapp
+```
+
+will copy the contents of `public/` to `/var/local/public`, rather than copying
```

Comment:
This doesn't quite describe what I'm seeing, which is that `whippet deploy -p /var/local/myapp` results in a public directory at `/var/local/myapp/public/`, as the app lives in `/var/local/myapp/current/` (symlinked to the appropriate folder in `/var/local/myapp/releases/`).

Reply:
Hmm, this is what I'm seeing:

```
❯ ls
total 0
drwxr-xr-x@ 6 snim2  wheel   192B 10 Jun 21:34 releases/
lrwxr-xr-x@ 1 snim2  wheel    74B 10 Jun 21:34 current -> /private/tmp/whip-deploy/releases/4451fc0e70d9cb8952482405b50150a8c8ff21eb
drwxr-xr-x@ 3 snim2  wheel    96B 10 Jun 21:34 public/
drwxr-xr-x@ 4 snim2  wheel   128B 10 Jun 21:18 shared/
```

but the `public/` directory lives in the app repo too, so maybe the question is should we delete it there when we deploy with `-p`?

Reply:
Yes, that matches what I'm seeing - but doesn't that mean that `whippet deploy -p /var/local/myapp` gives you a public folder at `/var/local/myapp/public`, not at `/var/local/public` as the docs claim (as all the folders listed there live inside `myapp/`)?